src/App.js

```diff
@@ -1242,6 +1241,42 @@ function runGameTick(initialGameState) {
     }
   });
 
+  /* 
+  Sandstorm
+
+  Start at 30 seconds, at 1 dmg
+  Tickrate is every 0.2 seconds
+  Increase by 1 every tick
```

Comment:
Formula doesn't look correct. For the first few ticks it seems to stay at 1 and then start ramping by 2 each time.

https://youtu.be/IurqE_Egvr0?si=MO6sq4XrBfmVDSov&t=61